src/amo/components/AddonBadges/styles.scss

```diff
@@ -3,15 +3,20 @@
 @import "~ui/css/vars";
 
 .AddonBadges {
-  @include respond-to(medium) {
+  width: 100%;
+
+  @include respond-to(large) {
     display: flex;
+    flex-flow: row wrap;
```

Comment:
Thanks, I meant to add this and maybe I forgot or was gonna do it in a follow-up. Much appreciated! 👍